pkg/helpers/imageregistry/client.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package imageregistry
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+)
+
+const (
+	// ClusterImageRegistryLabel value is namespace.managedClusterImageRegistry
+	ClusterImageRegistryLabel = "open-cluster-management.io/image-registry"
+
+	// ClusterImageRegistriesAnnotation value is a json string of ImageRegistries
+	ClusterImageRegistriesAnnotation = "open-cluster-management.io/image-registries"
+)
+
+type Registries struct {
```

Comment:
Does this only represent only one registry or many registries?

Reply:
only one registry. yeah it is confused. let me modify the name.

Reply:
modify to Registry